python/lsst/fgcmcal/fgcmFitCycle.py

```diff
@@ -231,6 +231,18 @@ class FgcmFitCycleConfig(pexConfig.Config):
         dtype=int,
         default=50,
     )
+    deltaMagBkgOffsetPercentile = pexConfig.Field(
+        doc=("Percentile brightest stars on a visit/ccd to use to compute net "
+             "offset from local background subtraction."),
+        dtype=float,
+        default=0.25,
+    )
+    deltaMagBkgPerCcd = pexConfig.Field(
+        doc=("Compute net offset from local background subtraction per-ccd? "
```

Comment:
Interesting question.  For these per ccd or per visit type flags in fgcmcal I haven't been using the `do` prefix, since I think of it as something slightly different.  And I'd like to keep this consistent with the others rather than deprecate a whole pile.